guides/source/testing.md

```diff
@@ -438,6 +444,10 @@ You can run a single test by appending a line number to a filename:
 
     bin/rails test test/models/user_test.rb:27
 
+You can run multiple test with in a line range by appending the line range to a filename:
```

Comment:
This is not part of test/USAGE, so not correct atm. We need to update https://github.com/rails/rails/blob/27b12e9249572d267e9ebf9f64b5bb2c21070f0d/railties/lib/rails/commands/test/USAGE as well.

Reply:
Updated.